### PR TITLE
Adding Lightbits CSI drivers to the list

### PR DESCRIPTION
**Pull request by sagyvolkov-lb**
This pull request introduces the Lightbits Labs CSI plugin to the CSI driver list.

Thank you!

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [kaDalu](https://github.com/kadalu/kadalu) | `org.kadalu.gluster` | v0.3 | A CSI Driver (and operator) for GlusterFS | Persistent | Read/Write Multiple Pods | Yes | 
 [KaiXiangTech MegaBric](https://github.com/kaixiangtech/csi-driver-flexblock) | `flexblock.csi.kaixiangtech.com` | v1.5.0 | A Container Storage Interface (CSI) plugin for KaiXiangTech MegaBric Storage | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Expansion, Cloning
 [KumoScale Block Storage](https://github.com/KioxiaAmerica/kumoscale-csi) | `kumoscale.kioxia.com` | v1.0 | A Container Storage Interface (CSI) Driver for KumoScale Block Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Topology
+[Lightbits Labs](https://github.com/LightBitsLabs/los-csi) | `csi.lightbitslabs.com` | v1.2 | A Container Storage Interface (CSI) Driver for Lightbits Storage | Persistent | Read/Write Single Pod (in volumeMode FileSystem) Read/Write Multiple Pods (in volumeMode Block) | Yes | Raw Block, Snapshot, Expansion, Cloning
```
Comment (xing-yang) [inline book/src/drivers.md:83]: Column 3 is for supported CSI spec version.  I see it is 1.3 here? 
https://github.com/LightBitsLabs/los-csi/blob/master/go.mod#L6

Comment (sagyvolkov-lb) [reply book/src/drivers.md:83]: should it include all supported versions?

Comment (xing-yang) [reply book/src/drivers.md:83]: Yes.

Comment (sagyvolkov-lb) [reply book/src/drivers.md:83]: I think it will be safer to say we're support >=v1.2

Comment (xing-yang) [reply book/src/drivers.md]: You can't claim support if it has not been tested with later versions.  Maybe you can say "1.2, 1.3" here or just list 1.3 which is the latest you've tested with?

Comment (sagyvolkov-lb) [reply book/src/drivers.md]: I have changed it to the two version we tested, 1.2 & 1.3. Thx!

**Pull request merged**